aiida/common/hashing.py

```diff
@@ -160,19 +160,13 @@ def _single_digest(obj_type, obj_bytes=b''):
 
 @_make_hash.register(bytes)
 def _(bytes_obj, **kwargs):
-    """
-    Hash arbitrary binary strings (str in Python 2, bytes in Python 3).
-    For compat reason between Python 2 and 3, this gets the same hash-type
-    as for unicode in Python 2, resp. str in Python 3."""
+    """Hash arbitrary byte strings."""
     return [_single_digest('str', bytes_obj)]
```

Comment:
Hmm, I'm not sure if we might want to change the "type specifier" here to be bytes - specific, now that py2 / py3 compatibility is no longer a concern.